scripts/build.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+
```

Comment:
Could you add some usage notes here, as well as some info about who should use this script?
..perhaps with a warning that this will install stuff globally on your system using sudo.